package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "npm-scripts",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "private": true,
   "scripts": {
-    "scss": "node-sass -o dist/css assets/scss/style.scss",
+    "scss": "sass assets/scss/style.scss dist/css/style.css",
     "concat:css": "concat -o dist/css/styles.css dist/css/style.css assets/css/test.css",
     "concat:js": "mkdir -p dist/js && concat -o dist/js/scripts.js assets/js/test1.js assets/js/test2.js",
     "concat": "npm run concat:css && npm run concat:js",
     "cssmin": "cleancss -o dist/css/styles.min.css dist/css/styles.css",
     "uglify": "uglifyjs -o dist/js/scripts.min.js dist/js/scripts.js",
-    "imagemin": "imagemin --out-dir=dist/img assets/img/**/*.{png,jpg,gif}",
+    "squoosh": "squoosh-cli --mozjpeg '{quality:70}' assets/img -d dist/img",
     "build:css": "npm run scss && npm run concat:css && npm run cssmin",
     "build:js": "npm run concat:js && npm run uglify",
     "build:img": "npm run imagemin",
```

Comment:
Should this be changed to `npm run squoosh`?

Reply:
Thanks, I picked this up as well.